### DISCUSSION TITLE
Troubleshooting Azure SAML Integration in Kimai

### DISCUSSION THREAD

**Discussion by BelCore-IT** [kimai/kimai#4948] [Questions & Answers]
Hello Everyone

I installed Kimai on a test environment using Docker-Compose, and the installation was successful. Now, I wanted to set up Azure SAML as described on the Kimai website. I followed all the steps meticulously and cleared the cache, but nothing happens. The login window still requires local credentials for logging in.

What am I doing wrong?

**Comment by kevinpapst** [comment]
Show you config and the steps you did.

**Comment by BelCore-IT** [comment]
Hi Kevin, I found the problem. I accidentally placed it in the wrong directory. Now I can see the button, but when I try to log in, the following error message appears: "An authentication exception occurred.

**Reply by kevinpapst** [reply]
Config invalid on Azure or Kimai. Hard to get right in the first try, usually needs some debugging.
Check our docker logs, hopefully there is something logged.
